MMM-Spotify.js

```diff
@@ -313,8 +393,7 @@ Module.register("MMM-Spotify", {
     title.textContent = playbackItem.name
 
     const album = document.querySelector("#SPOTIFY_ALBUM .text")
-    album.textContent = playbackItem.album.name
-
+    album.textContent = this.enableMiniBar ? (this.config.miniBarAlbum ? "- " + playbackItem.album.name + " -" : "-") : playbackItem.album.name
```

Comment:
I dont think youll need this (see comment below)

Reply:
i think you force `minimalistBar` style ... that's why it's not working
retry with position module `top_bar` or `bottom_bar`